anime-archive/src/pages/Closerlook/Closerlook.js

```diff
@@ -0,0 +1,68 @@
+import { closerlookData } from "../../utils/closerlookData";
+import BackToTop from "../../components/backToTop/BackToTop.js";
+import { Footer } from "../../components/footer/Footer.js";
+import "./Closerlook.css";
+
+export default function Closerlook() {
+  const data = closerlookData[0];
+  return (
+    <div>
+      {/* Title and Banner */}
+      <div className="closerlookBg">
+        <img src={data.bannerImage} alt="anime banner" />
+        <div className="closerlookHeroText">
+          <h3>{data.title.english}</h3>
+        </div>
+      </div>
+
+      <BackToTop />
+
+      {/* Top floating data card */}
+      <div className="closerlookFloat">
+        <div className="closerlookCover">
+          <img src={data.coverImage.large} alt="anime cover" />
+          <div className="closerlookStats">
+            <h4>Format</h4>
+            <p>{data.format}</p>
+            <h4>Episodes</h4>
+            <p>{data.episodes}</p>
+            <h4>Status</h4>
+            <p>{data.status}</p>
+            <h4>Studios</h4>
+            <p>{data.studios.nodes[0].name}</p>
+          </div>
+        </div>
+
+        <div className="closerlookDetails">
+          <h4>Genre</h4>
+          <p>{data.genres.join(", ")}</p>
+
+          {/* Character giant button here */}
+
+          <br />
+
+          <h4>Description</h4>
+          <p>{data.description}</p>
+
+          <br />
+
+          <div className="trailer">
+            {data.trailer ? (
```

Comment:
Trailer button does not show up in the preview deployment.

Reply:
Yeah just looked into this and I misused opacity: percent not needed and range is 0.0 to 1.0